src/main/java/org/takes/rq/RqLive.java

```diff
@@ -95,29 +97,16 @@ private static Request parse(final InputStream input) throws IOException {
                     break;
                 }
                 data = new Opt.Single<Integer>(input.read());
-                if (data.get() != ' ' && data.get() != '\t') {
-                    head.add(new String(baos.toByteArray()));
-                    baos.reset();
-                }
+                addHeader(data, baos, head);
```

Comment:
@yegor256 saw the difference. But don't you think that it should rather be:
`head = withHeader(data, baos, head);`
Since I'am asking to populate the `head` collection ?